IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -267,51 +397,56 @@ def describe_screen(screen_id):
 screen_details_df.to_csv(output_file, index=False, sep="\t")
 
 
-# In[13]:
+# In the chunk below, we collect and process a subset of the available IDR screens. The subset contains IDR accession names idr0080, idr0001, and idr0069. The idr0080 dataset was collected by Way et al. (2021) and will be used as a validation set for our statistics pipeline as we already know the image counts and unique attribute counts. The other two datasets are used for prototyping iterative functionalities of metadata collection and statistics.
```

Comment:
Long comments like these should be split into multiple lines

Reply:
The comment was split into multiple lines.

Added in dc9927a2e0ab6c5d2e75796145781bbf546c2431.